spec/features/view_post_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'pg'
+
+feature 'view posts' do
```

Comment:
good cleaning of the database. maybe could be in the spec_helper so you don't have to do it in every feature test? I may be misremembering that you can do that so disregard if so.